modules/oadp-restore-single-vm-from-multiple-vm-backup.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-restore-single-vm-from-multiple-vm-backup_{context}"]
+= Restoring a single VM from a backup of many VMs
+
+If you have a backup containing many virtual machines (VMs), and you want to restore only one VM, you can use the `LabelSelectors` section in the `Restore` CR to select the VM to restore. To ensure that the Persistent Volume Claim (PVC) attached to the VM is correctly restored, and the restored VM is not stuck in a `Provisioning` status, use both the `app: <vm_name>` and the `kubevirt.io/created-by` labels. To match the `kubevirt.io/created-by` label, use the UID of the `DataVolume` of the VM.
+
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have labeled the VMs that need to be backed up.
+* You have a backup of many VMs.
+
+.Procedure
+
+. Before you take a backup of many VMs, ensure that the VMs are labeled by running the following command:
++
+[source, terminal]
+----
+$ oc label vm <vm_name> vm-name=<vm_name> -n openshift-adp 
+----
+
+. Configure the label selectors in the `Restore` CR as shown in the following example:
++
+.Example `Restore` CR
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: singlevmrestore
+  namespace: openshift-adp
+spec:
+  backupName: multiplevmbackup
+  restorePVs: true
+  LabelSelectors:
+    - matchLabels:
+        kubevirt.io/created-by: <datavolume_uid> # <1>
```

Comment:
Is this created-by label also needed for the single-VM backup above?

Reply:
yes, so it restores the PVCs
and this is the reason i think RIA should handle it (it currently only restores the datavolume associate with the vm and not the pvc

Reply:
Yeah, I don't think the RIA functionality will override the include/exclude filtering for the restore.